<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QDialog" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>huis</string>
  </property>
  <widget class="QPushButton" name="kamer2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>261</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>kamer 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kamer3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>261</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string>kamer 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kamer1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>261</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string>kamer 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="camera">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>801</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="alarm">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>801</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Alarm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
